.pipelines/kubernetes.json

```diff
@@ -0,0 +1,46 @@
+{
+    "apiVersion": "vlabs",
+    "properties": {
+        "orchestratorProfile": {
+            "orchestratorType": "Kubernetes",
+            "orchestratorRelease": "$AKSENGINE_K8S_VERSION",
+            "kubernetesConfig": {
+                "loadBalancerSku": "Standard",
+                "excludeMasterFromStandardLB": true,
+                "addons": [
+                    {
+                        "name": "csi-secrets-store",
+                        "enabled": false
```

Comment:
If we are not using the addon in aks-engine, should we remove this whole section?

Reply:
`csi-secrets-store` addon is enabled by default for 1.16+ so, we are explicitly disabling the `csi-secrets-store` addon so we can install it as part of the e2e tests.